x-pack/plugins/security_solution/server/endpoint/routes/resolver/tree/utils/fetch.ts

```diff
@@ -226,7 +226,8 @@ export class Fetcher {
     // if the ancestry was specified then ignore the levels
     while (
       numNodesLeftToRequest > 0 &&
-      (options.schema.ancestry !== undefined || levelsLeftToRequest > 0)
+      options.schema.ancestry !== undefined &&
+      levelsLeftToRequest > 0
```

Comment:
@kqualters-elastic `levelsLeftToRequest` was used as a fallback when `schema.ancestry` is not available. I wonder if this will break requests that don't have `schema.ancestry`?